src/understand-tidb/the-lifecycle-of-a-statement.md

```diff
@@ -1 +1,200 @@
 # The Lifecycle of a Statement
+
+## MySQL protocol package with command and statement string
+
+After connecting and getting authenticated, the server is in a [statement execution loop](https://github.com/pingcap/tidb/blob/30cf15a59db11c34ffe05fc926152a43327eaa61/server/conn.go#L938) until the client is disconnected.
+
+The [dispatch](https://github.com/pingcap/tidb/blob/30cf15a59db11c34ffe05fc926152a43327eaa61/server/conn.go#L1111) function checks what kind of command was sent through the MySQL protocol and dispatches the matching function, like this snippet:
+```golang
```

Comment:
Here we use `golang`, almost we use `go` and so does below. Please keep consistency.

Reply:
```suggestion

```golang
```

ditto others

Reply:
Thanks for the sharp eye!